### PR TITLE
release in one step leveraging goreleaser

### PR DESCRIPTION
**Pull request by hilary**
see also #516 

Removed dockerhub integration, will submit that separately

CODE NOW RELEASES TO PROD TARGETS!

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -116,4 +73,91 @@ docker_build: _base_docker_cntr
...
+# ORIGIN ?= origin
+ORIGIN ?= hilary
+
+BUMP ?= patch
+
+.PHONY: version
+version:
+	@echo "==> determining doctl version"
+	@echo ""
+	@ORIGIN=$$ORIGIN scripts/version.sh
```
Comment (bentranter) [inline Makefile]: This syntax didn't work for me during my local testing, I had to use,

```Makefile
        @ORIGIN=${ORIGIN}
```

And then it all worked great.

File: Makefile [hunk 2]
```diff
@@ -116,4 +73,91 @@ docker_build: _base_docker_cntr
...
+_install_sembump:
+	@echo "==> installing/updating sembump tool"
+	@echo ""
+	@GO111MODULE=off go get -u github.com/jessfraz/junk/sembump
+
+.PHONY: _bump_and_tag
+_bump_and_tag: _install_sembump
+	@echo "==> BUMP=${BUMP} bumping and tagging version"
+	@echo ""
+	@ORIGIN=$$ORIGIN scripts/bumpversion.sh
```
Comment (bentranter) [inline Makefile]: Same here, I had to use,

```Makefile
        @ORIGIN=${ORIGIN}
```

File: Dockerfile.goreleaser [hunk 3]
```diff
@@ -0,0 +1,9 @@
+FROM alpine:3.8
+
```
Comment (bentranter) [inline Dockerfile.goreleaser]: Adding,

```Dockerfile
RUN apk add --no-cache curl
```

here fixes the issue with missing certs, from my testing at least.

Comment (hilary) [reply Dockerfile.goreleaser]: really? That's great!

**Pull request merged**